Dockerfile

```diff
@@ -15,6 +19,10 @@ COPY internal/ internal/
 COPY config/ config/
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build main.go
+#FROM builder as build
+RUN go install github.com/onsi/ginkgo/v2/ginkgo@v2.5.0
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -mod=mod main.go
+
+RUN go install
```

Comment:
I think this `go install` is unnecessary. I tested running the tests and main without it.

Reply:
Just made the changes!

Thanks!